galaxy_ng/tests/unit/api/test_api_ui_distributions.py

```diff
@@ -82,7 +82,7 @@ def test_distribution_list(self):
             self.client.force_authenticate(user=self.user)
             data = self.client.get(self.distro_url).data
 
-            self.assertEqual(len(data['data']), 6)
+            self.assertEqual(len(data['data']), 7)
             # 5 distros created by data migration and one `other-repo` created
```

Comment:
Minor nit for future testers, can we update this comment to note 6 created distros via data migration?